packages/react/src/Pagination/Pagination.client.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Pagination } from "./Pagination.client";
+
+const meta: Meta<typeof Pagination> = {
+  component: Pagination,
+  title: "Client/Components/Pagination",
+  parameters: {
+    layout: "centered",
+  },
+};
+export default meta;
+
+type StoryProps = Omit<
+  React.ComponentProps<typeof Pagination>,
+  "numberPage"
+> & {
+  numberPage: number;
+  onChangePage: (page: number) => void;
+  defaultPage?: number;
+};
+type Story = StoryObj<StoryProps>;
+
+export const Playground: Story = {
+  name: "Pagination",
+  render: ({ numberPage, onChangePage, defaultPage }) => (
+    <Pagination
+      numberPage={numberPage}
+      defaultPage={defaultPage}
+      onChangePage={onChangePage}
+    />
+  ),
+  args: {
+    numberPage: 20,
+    defaultPage: 2,
+    onChangePage: () => {},
```

Comment:
Use like this for functions : https://github.com/AxaFrance/design-system/blob/30e64583c2ff5bd0e1b33111c2230a30d95d6821/packages/react/src/Action/Action.stories.tsx#L31